### PR TITLE
Update master from integration

### PR DESCRIPTION
**Pull request by casthewiz**
Changeset for various edits to github actions, and bumping parser versions.

### CODE REVIEW

File: .github/workflows/deploy-integration-ec2.yml [hunk 1]
```diff
@@ -22,10 +22,10 @@ jobs:
           cd docs-worker-pool
           git fetch origin
           git checkout origin/integration
-          sudo docker system prune
+          sudo docker system prune -f
```
Comment (madelinezec) [inline .github/workflows/deploy-integration-ec2.yml:25]: can we please do -af 

(-a removes all unused images not just dangling ones)

File: .github/workflows/deploy-production-ec2.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ jobs:
           cd docker/docs-worker-pool
           git fetch origin
           git checkout $(git describe --tags)
-          sudo docker system prune
+          sudo docker system prune -f
```
Comment (madelinezec) [inline .github/workflows/deploy-production-ec2.yml:25]: same here

File: .github/workflows/deploy-production-ec2.yml [hunk 3]
```diff
@@ -45,7 +45,7 @@ jobs:
...
```
Comment (madelinezec) [inline .github/workflows/deploy-production-ec2.yml:48]: and here

File: .github/workflows/deploy-integration-ec2.yml (hunk 1 above)
Comment (casthewiz) [inline .github/workflows/deploy-integration-ec2.yml:25]:
```suggestion
          sudo docker system prune -af
```

File: .github/workflows/deploy-production-ec2.yml (hunk 2 above)
Comment (casthewiz) [inline .github/workflows/deploy-production-ec2.yml:25]:
```suggestion
          sudo docker system prune -af
```

File: .github/workflows/deploy-production-ec2.yml (hunk 3 above)
Comment (casthewiz) [inline .github/workflows/deploy-production-ec2.yml:48]:
```suggestion
          sudo docker system prune -af
```

**Review by madelinezec — APPROVED** [review]
let's squash upon merging to merge in fewer commits, but looks good!!

**Pull request merged**